docs/usage.rst

```diff
@@ -120,6 +120,15 @@ pseudo-elements::
     >>> selector.css('title::text').get()
     'Example website'
 
+Extract text witout ::text
+==========================
+You can extract inner text without specifying ``::text`` in your selctor instead
+an optional paramter text=True in the ``get()`` or ``getall()`` methods. 
+
+    >>> selector.css('title').get(text=True)
```

Comment:
Selecting the text of the `images` div may be a better illustration on how this is more powerful than `::text`:

```suggestion
    >>> selector.css('#images').get(text=True)
```

It should also be mentioned, before or after the example, how `text=True` is better than `::text`.

Reply:
Changed